test/unit/dialects/abstract/query-generator.test.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const chai = require('chai'),
+  expect = chai.expect,
+  Op = require('../../../../lib/operators'),
+  QueryGenerator = require('../../../../lib/dialects/abstract/query-generator');
+
+require(__dirname + '/../../support');
+
+function quoteIdentifier(identifier) {
+  return identifier;
+}
+
+function getQueryGenerator(sequelize) {
```

Comment:
All the other tests for the dialects query-generator loads them manually.
I think it also makes more sense in this context of a unit test for the abstruct query generator to get a clean and base version of query-generator since the one sequelize is using changes depending on the configuration it was loaded with and the dialect

Reply:
I have two suggestions here,

1) Keep this method here, but get QueryGenerator, directly with `sequelize.dialect.QueryGenerator`, no assign
2) Move this method to support lib, so others can use it as well

Reply:
Should I change it in all the other tests for each dialect query generator ?

Reply:
No just this one is enough